robot/Cumulus/tests/browser/recurring_donations/zedit_effective_date_erd_record.robot

```diff
@@ -14,13 +14,15 @@ Suite Teardown  Delete Records and Close Browser
 
 *** Keywords ***
 Setup Test Data
+     ${ns} =             Get NPSP Namespace Prefix
+     Set Suite Variable  ${ns}
```

Comment:
suggestion, change to capitols to be consistent

Reply:
same suggestion for all tests

Reply:
Updated it